src/components/map/map-gps-stations-layer.tsx

```diff
@@ -49,15 +53,80 @@ export class MapGPSStationsLayer extends BaseComponent<IProps, IState> {
         />
       );
     });
+    const mapScaleAdjust = 50;
+
+    const velocityArrows = visibleGPSStations.map(stat => {
+      // map scale is in meters per pixel, so use this as the basis for consistent length arrows at different zooms
+      // though the distances are so large we need to reduce the multiplier by a constant to fit the screen.
+      const arrowScale = mapScale / mapScaleAdjust;
+
+      const startLatLng = new LatLng(stat.latitude, stat.longitude);
+      // calculate the magnitude of the two component velocity values
+      const velMag = Math.sqrt((stat.eastVelocity ** 2) + (stat.northVelocity ** 2));
+      // Adjust the scale of the magnitude to improve visibility
+      // Stations move at a rate of meters per year, which is a small number.
+      // so we multiply by the map scale
+      const magnitude = velMag * arrowScale;
+      const dir = Math.atan(stat.northVelocity / stat.eastVelocity);
+
+      // we're working over a small area, so ignoring curvature of the Earth
+      const endLat = startLatLng.lat + (magnitude * Math.cos(dir));
+      const endLng = startLatLng.lng + (magnitude * Math.sin(dir));
+      const endLatLng = new LatLng(endLat, endLng);
+
+      // arrowhead
```

Comment:
Totally an aesthetic question, but it looks a little odd to me that the bigger arrows sometimes have huge arrow heads, and that also the width of the arrow head depends on the angle of the arrow (the more northerly, the tighter the arrow head).

If you calculate the points from the `endLatLng` instead of the start, it's easier to have a fixed-size head (with a maximum size depending on the magnitude, so you don't get arrow heads bigger than the vector). And if you add and subtract a little from the `dir`, instead of multiplying, you get constant angles.

Here's just a proposal, but again it's just aesthetics and feel free to dismiss:
```
const arrowBaseMag = Math.min(0.2, magnitude * 0.4);

const a1Lat = endLatLng.lat + (arrowBaseMag * -Math.cos(dir - 0.3));
const a1Lng = endLatLng.lng + (arrowBaseMag * -Math.sin(dir - 0.3));
const a1LatLng = new LatLng(a1Lat, a1Lng);

const a2Lat = endLatLng.lat + (arrowBaseMag * -Math.cos(dir + 0.3));
const a2Lng = endLatLng.lng + (arrowBaseMag * -Math.sin(dir + 0.3));
const a2LatLng = new LatLng(a2Lat, a2Lng);
```

Reply:
Arrowheads look much better - I'll add this in, thanks!